src/containers/SignUpPage.js

```diff
@@ -8,8 +8,13 @@ import { signUp } from 'actions/userActions';
 import SignUpForm from 'components/user/SignUpForm';
 import routes from 'constants/routesPaths';
 
-// Styles<
-import './SignUpPage.css';
+// Styles
+import './../styles/signUpPage.scss';
```

Comment:
As a convention for the scss files use dashes for the spaces.

For example: `./../styles/sign-up-case.scss`

Reply:
I fixed it.